shortames.conf

```
@@ -1,4 +1,10 @@
 [aliases]
+  # docker
+  "alpine" = "docker.io/alpine"
```

Will save some cycles normalizing it:

```suggestion
  "alpine" = "docker.io/library/alpine"
```